.github/workflows/generated-linux-aarch64-binary-manywheel-nightly.yml

```diff
@@ -37,7 +37,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  manywheel-py3_8-cpu-aarch64-build:
+  conda-py3_8-cpu-aarch64-experimental-split-build-build:
```

Comment:
This does not sound correct. why it has conda in the name ?
Also we should not touch linux-aarch64 builds for now. Let just test drive it in linux manywheel builds only for now